contracts/src/periphery/cert/EigenDACertTypes.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/core/libraries/v2/EigenDATypesV2.sol";
+
+/// @title EigenDACertTypes
+/// @notice This library defines the types for each EigenDA certificate version.
+library EigenDACertTypes {
+    struct EigenDACertV3 {
+        DATypesV2.BatchHeaderV2 batchHeader;
+        DATypesV2.BlobInclusionInfo blobInclusionInfo;
+        DATypesV1.NonSignerStakesAndSignature nonSignerStakesAndSignature;
+        bytes signedQuorumNumbers;
```

Comment:
can we add a comment explaining why we chose this field ordering

Reply:
https://github.com/Layr-Labs/eigenda/pull/1518/commits/c9a5699a6b126d274ecfffcdcc77ca208418cef3